backend/config/corpora-api.yml

```diff
@@ -630,6 +667,30 @@ components:
           type: string
         detail:
           type: string
+    gene_sets:
+      type: object
+      description: User created geneset information
+      properties:
+        gene_set_name:
+          type: string
+        gene_set_description:
```

Comment:
```suggestion
        description:
```

Reply:
Timmy is planning on using gene_set_name based on the tidy csv for simplicity

Reply:
Alright, please document somewhere why we are making an exception. I don't think we will remember why it is different from the rest of the APIs in a few months.